projects/plugins/jetpack/extensions/plugins/ai-content-lens/extend/ai-post-excerpt/index.tsx

```diff
@@ -2,6 +2,7 @@
  * External dependencies
  */
 import { useAiSuggestions } from '@automattic/jetpack-ai-client';
+import { useAnalytics } from '@automattic/jetpack-shared-extension-utils';
```

Comment:
Can we do the import on the same line of the other two, below, since they are all provided by `@automattic/jetpack-shared-extension-utils`?

Reply:
Good catch! Just pushed the change and included a missing type from ai-client